containers/ContractState.ts

```diff
@@ -38,7 +38,6 @@ const useContractState = () => {
 
   function updateState() {
     if (!contract || !signer) return;
-    setLoading(true);
```

Comment:
this was causing some views to show N/A when a refresh was happening to the data, so i removed it